include/tdi/common/tdi_init.hpp

```diff
@@ -172,7 +172,7 @@ class DevMgr {
 class Init {
  public:
   /**
-   * @brief TDI Module Init API. This function needs to be called to
+   * @brief Bf Rt Module Init API. This function needs to be called to
```

Comment:
Again change in comment and not in code. Btw please don't change to Bf_Rt because thats target specific and this part of the code is tdi generic.